lib/bundler/plugin/index.rb

```diff
@@ -95,6 +95,14 @@ def installed?(name)
         @plugin_paths[name]
       end
 
+      def installed_plugins
+        @plugin_paths.keys
+      end
+
+      def plugin_commands(plugin)
```

Comment:
Is this suppose to be `plugins` instead of `plugin`?

Reply:
ping @fotanus

Reply:
Sorry for the delay @colby-swandale, I wrote this PR 5 months ago, and I'm a bit out of what is happening on bundler. I think that on line 102 it should be `plugin`, and in line 103 also `plugin`, since the method appears to list the commands for one plugin. I'm actually unsure how this pass on the test? Right now it seems like a typo to me

Reply:
Are you still working on this PR?

Reply:
Feel free to take over or reimplement copying any ideas you want :-) I can't deal with this right now.